### PR TITLE
Add Avatar Component

### PR DESCRIPTION
**Pull request by varun13169**
Add Avatar Component
	- avatar html and css files.

### CODE REVIEW

**Comment by varun13169** [comment]
Continuing from the following PR https://github.com/varun13169/delta-ui/pull/2

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <style>
+        body {
+            padding: 0px;
+        }
+        .component-holder {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-end;
+        }
+        .comp-lib-hr {
```
Comment (iRohitGaur) [inline components/avatar/avatar.html]: you can merge two different classes using a comma if they have the same styling

File: library/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    position: relative;
+}
+.avatar__content {
+    width: 6.4rem;
+    height: 6.4rem;
+    border-radius: 50%;
+}
+
+/* Avatar Component Starts -- Extra Large */
```
Comment (iRohitGaur) [inline library/avatar/avatar.css:21]: same: you can merge two different classes using a comma if they have the same styling

**Review by iRohitGaur — CHANGES_REQUESTED** [review]
you can merge two different classes using a comma if they have the same styling

**Comment by archit7448** [comment]
> Continuing from the following PR #2
Resolved ,good to merge

File: components/avatar/avatar.html (hunk 1 above)
Comment (varun13169) [reply components/avatar/avatar.html]: Done.

File: library/avatar/avatar.css (hunk 2 above)
Comment (varun13169) [reply library/avatar/avatar.css:21]: Not Needed Anymore.

**Review by iRohitGaur — APPROVED** [review]

**Review by archit7448 — APPROVED** [review]

**Pull request merged**